metricshub-classloader-agent/pom.xml

```diff
@@ -26,7 +26,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>3.8.1</version>
+					<version>3.13.0</version>

```

Comment:
@CherfaElyes, should we continue to manage this plugin here? It is already managed in the main POM.